libmscore/textedit.cpp

```diff
@@ -663,6 +663,7 @@ void TextBase::inputTransition(EditData& ed, QInputMethodEvent* ie)
                   editInsertText(_cursor, preEdit);
                   setTextInvalid();
                   layout1();
+                  score()->update();
```

Comment:
This fixes the problem of pre-edit strings not being displayed immediately after each keystroke.